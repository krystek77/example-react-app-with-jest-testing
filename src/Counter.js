import React from 'react';

export default function Counter({ counter }) {
  return React.createElement('p', { className: 'Counter' }, counter);
  //   return <div className='Counter'>{count}</div>;
}
