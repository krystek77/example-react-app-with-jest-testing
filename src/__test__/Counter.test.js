import React from 'react';
import renderer from 'react-test-renderer';
import Counter from '../Counter';
describe('Counter', () => {
  it('snapshot renders', () => {
    const component = renderer.create(<Counter counter={1} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('snapshot renders 2', () => {
    const component = renderer.create(<Counter counter={1} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
