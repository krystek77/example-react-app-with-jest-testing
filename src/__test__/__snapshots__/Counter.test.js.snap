// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter snapshot renders 1`] = `
<p
  className="Counter"
>
  1
</p>
`;

exports[`Counter snapshot renders 2 1`] = `
<p
  className="Counter"
>
  1
</p>
`;
